colabs/Pierwsze_wnioski_z_data_setu.ipynb

```diff
@@ -0,0 +1,2689 @@
+              "          const dataTable =\n",
+              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
+              "                                                     [key], {});\n",
+              "          if (!dataTable) return;\n",
+              "\n",
+              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
+              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
+              "            + ' to learn more about interactive tables.';\n",
+              "          element.innerHTML = '';\n",
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ]
+          },
+          "metadata": {},
+          "execution_count": 39
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Tabele, która wydają się być najbardziej istotne w dalszej pracy bazą, to *title_basics* i *title_ratings*."
+      ],
+      "metadata": {
+        "id": "HvV_7gV4hXSo"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### \\N w title_basics i title_ratings"
+      ],
+      "metadata": {
+        "id": "Rn1a-UKHCAmv"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "N_basics_count = []\n",
+        "for i in range(len(title_basics.columns)):\n",
+        "  N_basics_count.append(title_basics.where(F.col(title_basics.columns[i]) == \"\\\\N\").count())\n",
+        "\n",
+        "N_ratings_count = []\n",
+        "for i in range(len(title_ratings.columns)):\n",
+        "  N_ratings_count.append(title_ratings.where(F.col(title_ratings.columns[i]) == \"\\\\N\").count())\n",
+        "\n",
+        "N_basics = pd.DataFrame(data = {\"nazwa_kolumny\":title_basics.columns, \"liczba wartości \\\\N\":N_basics_count})\n",
+        "\n",
+        "N_ratings = pd.DataFrame(data = {\"nazwa_kolumny\":title_ratings.columns, \"liczba wartości \\\\N\":N_ratings_count})\n",
+        "\n",
+        "display(N_ratings)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 458
+        },
+        "id": "LMeffaH8CIVb",
+        "outputId": "88daf0dd-9be0-4fca-8791-13c1cc96f678"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "    nazwa_kolumny  liczba wartości \\N\n",
```

Comment:
założyć zadanie na usunięcie tych pseudo-nulli z datasetu, a tu dopisać TODO by to usunąć po rozwiązaniu zadania (wkleić do niego link)

Reply:
https://app.clickup.com/t/334er77

Reply:
ale brakuje TODO w notatniku

Reply:
dopisałem